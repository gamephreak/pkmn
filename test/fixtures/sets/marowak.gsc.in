Marowak (M) @ Leftovers  
- Earthquake  
- Hidden Power [Bug]  
- Rock Slide  
- Swords Dance  
