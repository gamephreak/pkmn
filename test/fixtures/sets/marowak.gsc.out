Marowak (M) @ Leftovers  
EVs: 252 HP / 252 Atk / 252 Def / 252 SpA / 252 SpD / 252 Spe  
IVs: 26 Atk / 26 Def  
- Earthquake  
- Hidden Power [Bug]  
- Rock Slide  
- Swords Dance  

